examples/Makefile

```diff
@@ -1,13 +0,0 @@
-CXXFILES = test1.cpp ../pipeline.cpp
```

Comment:
Ty examples tam byly myslene nejen jako test, ale i jako soucast dokumentace. Jako ze "takhe vypada typicke pouziti". Mozna je na tohle ale stejne brzo, protoze se to api bude jeste vyvijet...

Reply:
On je ten rozsireny example zaroven pouzity v jednom testu, ale to pak neni explicitne receno, ze je to pouzitelne i jako example. Mam to teda vratit?

Reply:
hlavni nevyhoda examplu je ze se musi nejakym zpusobem spoustet dohromady s testy, jinak se rozpadnou od aktualniho kodu. Asi to zatim nech tak jak to je, dodelame je kdyztak potom

Reply:
Ok